src/rebuild/srv.c

```diff
@@ -1263,7 +1287,8 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 		 * reclaim the current rebuilding fails.
 		 */
 		if (rgt->rgt_init_scan) {
-			rc = ds_rebuild_schedule(pool, task->dst_map_ver, rgt->rgt_stable_epoch,
+			rc = ds_rebuild_schedule(pool, task->dst_reclaim_ver - 1,
```

Comment:
could you please remind why need "task->dst_reclaim_ver - 1" here? (or add a few comment if you need to refresh the patch), thanks.

Reply:
oh, reclaim_version is the minimum version of all rebuilding targets, so once the rebuild fail, it will be used to discard all the previous rebuild, but keep the inflight I/O part, so either we use -1 here or -1 in obj_reclaim.  Thanks.